### PR TITLE
print pipeline run info when pipeline run doesn't finish successfully

### PR DESCRIPTION
**Pull request by rcerven**
* CLOUDBLD-8220

Signed-off-by: Robert Cerven <rcerven@redhat.com>

# Maintainers will complete the following section

- [ ] Commit messages are descriptive enough
- [ ] Code coverage from testing does not decrease and new code is covered
- [ ] JSON/YAML configuration changes are updated in the relevant schema
- [ ] Changes to metadata also update the documentation for the metadata
- [ ] Pull request has a link to an osbs-docs PR for user documentation updates

### CODE REVIEW

File: osbs/cli/main.py [hunk 1]
```diff
@@ -211,6 +214,12 @@ def _display_pipeline_run_summary(build_metadata):
         output.append("")
         output.append(error_msg)
 
+    pipeline_run_info = results['info']
+    if pipeline_run_info:
+        output.append("")
+        output.append("pipeline run info:")
```
Comment (MartinBasti) [inline osbs/cli/main.py:220]: why you don;t just return json?

Comment (MartinBasti) [reply osbs/cli/main.py:220]: nevermind

**Review by ben-alkov — APPROVED** [review]

File: osbs/tekton.py [hunk 2]
```diff
@@ -540,7 +540,9 @@ def get_error_message(self):
         return err_message
 
     def has_succeeded(self):
-        return self.status_reason == 'Succeeded'
+        status_reason = self.status_reason
+        logger.info("Pipeline run info: '%s'", self.data)
```
Comment (MartinBasti) [inline osbs/tekton.py:544]: .debug right?

**Review by MartinBasti — APPROVED** [review]

File: osbs/tekton.py (hunk 2 above)
Comment (rcerven) [reply osbs/tekton.py:544]: koji-c doesn't print debug

**Review by ben-alkov — APPROVED** [review]

**Pull request merged**